lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -383,7 +383,7 @@ def post_data(params)
         return nil unless params
 
         params.map do |key, value|
-          next if value.nil?
+          next if value != false && value.blank?
```

Comment:
Won't this call `next` on all `values` except `false` now? How can any other value be included in the request now other than `false`?